docs/program/apis/sessions.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Session Management with Viam's Client SDKs"
+linkTitle: "Session Management API"
+weight: 20
+type: "docs"
+description: "How to use the session management API with Viam's Client SDKs."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session", "sessions", "session management"]
+---
+
+The session management API provides support for robot session management.
+Normally, sessions are managed for you with default resource configuration, but if you want to manage sessions yourself, you can with Viam's sessions management API.
+
+A session:
+
+- allows a client to express that it is actively connected or authenticated to `viam-server` on your robot
+- supports stopping moving components when the connection is no longer active.
+
+### Purpose of session management
+
+When controlling a robot or fleet with Viam, you want a way to understand the presence of the clients that are communicating with and authenticated to each robot's `viam-server`.
+The period of time during which these clients are present is called a *session*.
+Imagine a session as analogous to the lifetime of a robot, in regards to its network connection.
+*Session management* is a presence mechanism between a client and `viam-server`, or, a way to manage the lifetime of the robot yourself.
+
+Session management allows for safer operation of robots that physically move.
+For example, imagine a wheeled rover gets a [`SetPower()`](/components/base/#setpower) command as the last input from a client before the connection to the robot is interrupted.
+Without session management, the API request from the client sets the flow of electricity to the motors of the robot and then does not time out, causing the robot to continue driving forever, colliding with objects and people.
+
+With session management, if the robot does not receive a signal from the client at regular intervals, it safely stops until the connection is reestablished.
```

Comment:
```suggestion
With session management, if the robot does not receive a signal from the client at regular intervals, it safely stops until the connection is reestablished.

```
This is a separate idea from the next few sentences because those are talking about how client can be various things but this isn't really about that even though it has the word "client" in it.

Reply:
Ok cool done!